sdk/servicebus/service-bus/src/models.ts

```diff
@@ -130,10 +130,10 @@ export interface MessageHandlerOptions {
    * by the sdk automatically. This auto renewal stops once the message is settled or once the user
    * provided onMessage handler completes ite execution.
    *
-   * - **Default**: `300` seconds (5 minutes).
+   * - **Default**: `300 * 1000` milliseconds (5 minutes).
```

Comment:
A few lines above this line in the docs, we are refering to seconds

Reply:
Updated.